package.json

```diff
@@ -3,7 +3,7 @@
     "version": "10.0.11",
     "description": "Test plugin to lern dev enveroument",
     "scripts": {
-        "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
+        "build": "cd ./ngx-flow && npm run build:component && cd .. && webpack -c ./.config/webpack/webpack.config.ts --env production",
```

Comment:
I would propose splitting this in two scripts we can call in sequence ie: `build-ngx` and `build` rather than mixing them together and then in the action `npm run build-npx && npm run build`